v22.2/crdb-internal.md

```diff
@@ -873,6 +873,22 @@ Column | Type | Description
 `aggregation_interval` | `INTERVAL NOT NULL` | The interval over which statistics are aggregated.
 `index_recommendations` | `STRING[] NOT NULL` | An array of strings containing [index recommendations](ui-insights-page.html#schema-insights-tab) of the format `{type} : {sql query}`.
 
+#### `fingerprint_id` column
+
+The value is in hexadecimal format. The following examples show how to use it to query `statement_statistics`.
```

Comment:
```suggestion
The value is in hexadecimal format. The following examples show how to use this column to query `statement_statistics`:
```

Reply:
Thanks for the suggestions! Instead of `this column` it would be more accurate to use `this value`.

Reply:
updated to use `this value` and colon.